testing/perl-db_file-lock/APKBUILD

```diff
@@ -16,14 +16,13 @@ depends="$cpandepends"
 makedepends="perl-dev perl-db_file $cpanmakedepends"
 checkdepends="$cpancheckdepends"
 subpackages="$pkgname-doc"
-source="https://cpan.metacpan.org/authors/id/D/DH/DHARRIS/DB_File-Lock-0.05.tar.gz"
+source="https://cpan.metacpan.org/authors/id/D/DH/DHARRIS/DB_File-Lock-0.05.tar.gz $pkgname.patch"
```

Comment:
Please move the patch to a new line, this makes it more obvious for others about the inclusion of the patch.
Also rename the patch to what it actually tries to fix, or something useful instead of the pkgname.
You can also squash these commits into a single commit and add a message about the fixes in this commit.

```
source="https://cpan.metacpan.org/authors/id/D/DH/DHARRIS/DB_File-Lock-0.05.tar.gz
    fix-whatever.patch"
```